__tests__/controllers/auth/rate-limit-endpoint.spec.ts

```diff
@@ -0,0 +1,23 @@
+import request from 'supertest';
+import app from '@app';
+import { API } from '../../utils';
+import { RATE_LIMIT_MAX_REQUESTS } from '@config';
+describe('calling multiple times an endpoint', () => {
```

Comment:
```suggestion
import { RATE_LIMIT_MAX_REQUESTS } from '@config';

describe('calling multiple times an endpoint', () => {
```

just to be consistent with the rest

Reply:
thanks for advising, changed!